### PR TITLE
Remove Weave Cloud mentions

### LINKED ISSUE #2180: Remove Weave Cloud mentions in docs
In the discussions about Flux's [application for CNCF Sandbox](https://github.com/cncf/toc/pull/232) the need to decouple Flux from Weave Cloud to some degree was discussed.

Historically the development of Flux and Weave Cloud have been quite intertwined. In the future we will want to put all integrations with other products and projects on a level playing field.

Therefore I suggest we explain in our docs what historically the connection between Flux and Weave Cloud was, move some of the current docs to Weave Cloud Deploy docs and link from our FAQ or Troubleshooting guide, so that if people get stuck with the Flux portion of Weave Cloud, they know where to find help.

Here's what I dug out with a quick `git grep` (there might be more - I'll double-check):

```
README.md:- [Weave Cloud](https://cloud.weave.works)
README.md:### Relation to Weave Cloud
README.md:Weave Cloud is a SaaS product by Weaveworks that includes Flux, as well
README.md:If you want to learn more about Weave Cloud, you can see it in action on
api/api.go:// to Weave Cloud.
api/api.go:// Weave Cloud.
chart/flux/README.md:#### To connect Flux to a Weave Cloud instance:
chart/flux/README.md:| `token`                                           | `None`                                               | Weave Cloud service token
site/daemon.md:| --connect                                        |                          | connect to an upstream service e.g., Weave Cloud, at this base address
site/fluxctl.md:[Weave Cloud](https://cloud.weave.works).
site/fluxctl.md:  # To a Weave Cloud instance, with your instance token in $TOKEN
site/fluxctl.md:  -t, --token string                    Weave Cloud authentication token; you can also set the environment variable WEAVE_CLOUD_TOKEN or FLUX_SERVICE_TOKEN
site/fluxctl.md:[Weave Cloud](https://cloud.weave.works) by selecting the "Automate"
site/fluxctl.md:Actions triggered by a user through the Weave Cloud UI or the CLI `fluxctl`
site/helm-operator.md:## Installing Weave Flux Helm Operator for Weave Cloud
site/helm-operator.md:In order to use the Helm operator with Weave Cloud you have to apply the `HelmRelease` CRD definition and the operator
site/how-it-works.md:Flux provides a CLI ([`fluxctl`](fluxctl.md)) and a UI (as a component of Weave Cloud)
site/how-it-works.md:# Weave Cloud only
site/introduction.md:[Weave Cloud](https://cloud.weave.works) fixes this problem.
site/troubleshooting.md:If you're using [Weave Cloud](https://cloud.weave.works/), this
site/troubleshooting.md:from the settings in Weave Cloud; set the environment variable
site/troubleshooting.md:probably means Flux is defaulting to using Weave Cloud because you've
site/troubleshooting.md:output of `fluxctl list-images` (or the UI in Weave Cloud, if you're
site/upgrading-to-1.0.md:Weave Cloud. In version 1, the daemon does all of the mechanical
site/upgrading-to-1.0.md:work by itself, and Weave Cloud “merely” adds a web user interface and
site/upgrading-to-1.0.md: * [Connect Flux to Weave Cloud](#if-you-are-connecting-to-weave-cloud); or,
site/upgrading-to-1.0.md:   Weave Cloud).
site/upgrading-to-1.0.md:## <a name="weavecloud">If you are connecting to Weave Cloud</a>
site/upgrading-to-1.0.md:token (found in the Weave Cloud UI for the instance you are
site/upgrading-to-1.0.md:Run Weave Cloud’s launch generator to delete the resources in the
site/upgrading-to-1.0.md:To run the new Flux with Weave Cloud:
site/upgrading-to-1.0.md:To run Flux without connecting to Weave Cloud, adapt the manifests
site/upgrading-to-1.0.md:resources by name and delete them. Weave Cloud’s “Explore” tab may
site/upgrading-to-1.0.md:before running Flux v1. You can download manifests from Weave Cloud,
```

### PR DESCRIPTION
**Pull request by dholbach**
In order to fix #2180 I'm removing a bunch of WC related things - let me know what you think.

I'll move whatever's still relevant up to https://www.weave.works/docs/cloud/latest/tasks/deploy/ somewhere.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,19 +80,14 @@ Its major features are:
...
-- full observability and insights into your cluster: Instantly start using
-  monitoring dashboards for your cluster, hosted 13 months of history, use
-  a realtime map of your cluster to debug and analyse its state.
-
-If you want to learn more about Weave Cloud, you can see it in action on
-[its homepage](https://www.weave.works/product/cloud/).
+### History
+
+In the first years of its existence, the development of Flux was very
+closely couple to the one of [Weave
```
Comment (squaremo) [inline README.md]:
```suggestion
closely coupled to that of [Weave
```

File: site/introduction.md [hunk 2]
```diff
@@ -8,12 +8,6 @@ that surround building, deploying and monitoring applications. The
 goal is to provide a sustainable model for maintaining and improving 
 an application.
 
-The promise of continuous delivery relies upon automation and in recent 
-years the automation of building and testing software has become 
-commonplace. But it is comparatively difficult to automate the 
-deployment and monitoring of an application.
-[Weave Cloud](https://cloud.weave.works) fixes this problem.
-
 Weave Flux is a tool that automates the deployment of containers to
```
Comment (squaremo) [inline site/introduction.md]:
```suggestion
Flux is a tool that automates the deployment of containers to 
```

**Comment by dholbach** [comment]
Shall I move all mentions from "Weave Flux" to "Flux"?
```
[daniel@reef flux ]$ git grep -i "weave flux" | wc -l
43
[daniel@reef flux ]$ 
```

**Comment by dholbach** [comment]
Thanks for the reviews @squaremo and @stefanprodan so far!

**Comment by dholbach** [comment]
I left some "Weave Flux" mentions in the code, as it might break email filters(?)... LMK if you still want to change them.

**Comment by dholbach** [comment]
Remaining occurrences of "weave flux" are:

```
chart/flux/README.md:| `git.user`                                        | `Weave Flux`                                         | Username to use as git committer
chart/flux/values.yaml:  user: "Weave Flux"
cmd/fluxd/main.go:              gitUser      = fs.String("git-user", "Weave Flux", "username to use as git committer")
daemon/sync_test.go:    gitUser     = "Weave Flux"
gpg/gpgtest/gpg.go:     io.WriteString(stdin, "Name-Real: Weave Flux\n")
site/daemon.md:| --git-user                                       | `Weave Flux`             | username to use as git committer
site/fluxctl.md:| git-user       | committer name             | `Weave Flux`
```

Please advise on how to proceed (or feel free to just push to the branch).

**Comment by hiddeco** [comment]
@dholbach renamed 

```
daemon/sync_test.go:    gitUser     = "Weave Flux"
gpg/gpgtest/gpg.go:     io.WriteString(stdin, "Name-Real: Weave Flux\n")
```

as they are only used in tests.

**Comment by squaremo** [comment]
> cmd/fluxd/main.go:              gitUser      = fs.String("git-user", "Weave Flux", "username to use as git committer")

I've lodged an issue for this and `gitEmail`; you don't need to include them here.

**Comment by dholbach** [comment]
Want me to link https://www.weave.works/docs/cloud/latest/tasks/deploy/managing-flux/ from somewhere?

**Pull request merged**